src/main/scala/org/clulab/wm/eidos/groundings/grounders/SRLCompositionalGrounder.scala

```diff
@@ -99,10 +100,19 @@ class SRLCompositionalGrounder(name: String, domainOntology: DomainOntology, w2v
     groundingResult
   }
 
+  // FIXME: This could probably be implemented better by creating a real StopwordManager
+  // Everything after TIME is new to this grounder
+  val STOP_NER: Set[String] = Set("DATE", "DURATION", "LOCATION", "MISC", "MONEY", "NUMBER", "ORDINAL",
+    "ORGANIZATION", "PERSON", "PLACE", "SET", "TIME", "NATIONALITY", "COUNTRY", "TITLE", "STATE_OR_PROVINCE")
+
   override def groundEidosMention(mention: EidosMention, topN: Option[Int] = None, threshold: Option[Float] = None): Seq[OntologyGrounding] = {
     // Do nothing to non-groundable mentions
     if (!EidosOntologyGrounder.groundableType(mention))
       Seq(newOntologyGrounding())
+    // Do nothing to named entities
+    if (mention.odinMention.entities.get.toSet.intersect(STOP_NER).nonEmpty) {
```

Comment:
This looks suspect to me... is it saying that we are not grounding anything that has _any_ ners?

Reply:
It's definitely not *all* ners, but it is a non-short list. See [here](https://github.com/clulab/eidos/pull/937/files#diff-2924da707a0df24b68254e2f9234330df0757e0db36a0103229b520a3860bb43R105) for the specific line with them all listed.

Maybe my understanding was wrong, but I thought one thing we wanted to do was avoid grounding them because there are a bunch of things like woredas and provinces that we didn't want.

Reply:
we don't want to ground the locations themselves, and we don't want to include the names of the locations during grounding, but what is the mention here? If we had "Paris poverty" would we skip it completely bc there was an intersection (bc of Paris)?

Reply:
The mentions I saw were typically things like `Addis Ababa`, `South Sudan`, and `Minister Hailemariam`. For the first two, it makes sense to just skip the whole thing, but what about people with titles (or locations such as `Somali region`)? Should we keep the more generic word and only skip the ner piece? Or will that cause problems?